src/gennodejs/generate.py

```diff
@@ -818,6 +830,7 @@ def write_msg_types(s, spec):
 
         for field in fields:
             s.write('public {}: {};'.format(field.name, get_js_type(field, spec.package)))
+        write_type_constants(s,spec)
```

Comment:
we need this in msgs too, not just services.

Reply:
this is in msgs. But I will add it in services too